frontend/src/pages/Risk/Risk.tsx

```diff
@@ -568,8 +571,20 @@ const Risk: React.FC = (props) => {
       const imgWidth = 190;
       const imgHeight = (canvas.height * imgWidth) / canvas.width;
       const pdf = new jsPDF('p', 'mm');
-      pdf.addImage(imgData, 'PNG', 10, 10, imgWidth, imgHeight);
-      pdf.save('Crossfeed_Report.pdf');
+      const img_crossfeed =
```

Comment:
Can you add a comment noting what image this is? Is there a way to import the file through something like a base64-loader so we don't need to hard code it? (though if that's too cumbersome, we don't need to do that)

Reply:
I'm just going to drop the image.   It's the Crossfeed logo base64 encoded.  We can add it back it in later if desired.